css/wfrp4e.css

```diff
@@ -14724,6 +14724,7 @@ button.capture-position {
 .condition-chat,
 .chat-roll,
 .table-click,
+.travel-click,
```

Comment:
This is specifically for links that show up in chat/editors. Is this intended to work with links? How so?

Reply:
yes, when you issue this kind of  command :
/travel -> list of all possible departure towns
/travel town1 -> list of all possibles arrival towns

Reply:
Ah, I see the travel-click html now, thanks.